ql/src/java/org/apache/hadoop/hive/ql/ddl/table/storage/skewed/AlterTableNotSkewedDesc.java

```diff
@@ -38,6 +38,6 @@ public AlterTableNotSkewedDesc(TableName tableName) throws SemanticException {
 
   @Override
   public boolean mayNeedWriteId() {
-    return false;
+    return true;
```

Comment:
AFAIK we do not support skewed ACID tables

Reply:
Flipping it back to false.